### ISSUE TITLE
Design document for split screen chiventure with raylib graphic

### ISSUE THREAD

**Issue by perenewang** [uchicago-cs/chiventure#890]
Write a design document for displaying chiventure with a raylib graphic as the top half of the screen (building off of the split screen example)

Assignees: Gautam and Perene
Team: gui

**Comment by perenewang** [comment]
Created ui_enhancement.h and draw_images.h to create functions to include graphics in the split screen that correspond with the chiventure game. Particularly, we made draw_room and draw_object functions that will be called in the start_ui function when the user moves rooms or picks up an object.

**Comment by grkapoor17** [comment]
Following up on a discussion with our senior developer, Maxine, we have decided to delete the ui_enhancement.h file. Instead, we plan to update the ui.c file, which will now call Loading and Drawing Image functions from the draw_images.h module. We have also created a [wikipage](https://github.com/uchicago-cs/chiventure/wiki/Design-document-for-integrating-split-screen-graphics-with-chiventure),  that details our documentation and next steps going into the next sprint for our implementation.
